src/ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusModal.jsx

```diff
@@ -8,16 +9,34 @@ import { List, Row, Radio } from 'components/List'
 import iconReimbursement from 'assets/icons/icon-reimbursement-detailed.svg'
 import styles from 'ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusModal.styl'
 import { getReimbursementStatus, getLabel } from 'ducks/transactions/helpers'
+import { isHealthExpense } from 'ducks/categories/helpers'
+import flag from 'cozy-flags'
+import ContactItem from 'ducks/transactions/actions/ReimbursementStatusAction/ContactItem'
 
 class _ReimbursementStatusModal extends React.PureComponent {
   render() {
-    const { transaction, onChange, t, className, ...rest } = this.props
+    const {
+      transaction,
+      onChange,
+      t,
+      className,
+      breakpoints: { isMobile },
+      brands,
+      ...rest
+    } = this.props
     const choices = ['pending', 'reimbursed', 'no-reimbursement']
     const status = getReimbursementStatus(transaction)
 
+    const showContacts =
+      flag('reimbursements-contacts') && isHealthExpense(transaction)
+
     return (
-      <Modal mobileFullscreen className={cx('u-pt-2', className)} {...rest}>
-        <header className="u-ta-center">
+      <Modal
+        mobileFullscreen
+        className={cx('u-flex', 'u-flex-column', className)}
+        {...rest}
+      >
+        <header className="u-ta-center u-pt-2">
```

Comment:
I think it's the modal that should have a padding, not the header.

Reply:
You are right. I did this because there was a problem when setting the padding on the modal and I forgot to have a look at what was wrong. I did it: https://github.com/cozy/cozy-ui/pull/1069

I will restore the padding on the modal when this PR is merged.